sample_metadata/parser/sample_file_map_parser.py

```diff
@@ -23,13 +23,27 @@
 - 'Filenames'
 - ['Type']
 
-EG:
-    Sample ID   Filenames
+e.g.
+    Sample ID       Filenames
     <sample-id>     <sample-id>.filename-R1.fastq.gz,<sample-id>.filename-R2.fastq.gz
     # OR
     <sample-id2>    <sample-id2>.filename-R1.fastq.gz
     <sample-id2>    <sample-id2>.filename-R2.fastq.gz
 
+Example with optional columns
+Note: Individual ID column must contain values in every row
```

Comment:
Because it calls the participant upsert endpoint which has an entirely different structure. It's keyed on the participant name (external participant id). If we have samples without a participant we can use the batch sample upsert endpoint. 

I think it just makes it a bit cleaner. In a weird case where we have some samples and not others we would have two files. 

Otherwise, we can add a ticket to allow participant-less upsert of the samples without a participant. I'd add that as another feature request/ticket.